tests/integration/integration_test_case.py

```diff
@@ -22,78 +54,163 @@ def setUp(self):
             password = os.getenv(password_name, dev_default)
             vars(settings)[password_name] = password  # assigns attribute to this module
 
-        self.application = create_app()
-        self.client = self.application.test_client()
+    def _setUpApp(self):
+        self._application = create_app()
+        self._client = self._application.test_client()
+        # Clear any existing state before the test starts
+        self.clearDatabase()
```

Comment:
Because we init the database on import rather than setup it will contain data from other tests in the same test case. I thought each test should be run with a clean db so wipe it here.